docs/DialogActivityExport_Summary.md

```
@@ -0,0 +1,187 @@
+
+### Create All Indexes (DBA)
+```sql
+-- See docs/database/DBA_Index_Creation_Scripts.sql
+-- Copy-paste the 6 CREATE INDEX CONCURRENTLY statements
+```
+
+---
+
+## File Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)